### DISCUSSION TITLE
파이어베이스 연동

### DISCUSSION THREAD

**Discussion by hyeona-y** [naverz/zepeto-studio-kor#1159] [5. Q&A]
파이어베이스 연동해서 사용하려고 하는데 혹시 참고할 수 있는 자료나 관련 문서를 얻을 수 있을까요? 

도움 부탁드립니다ㅜㅜ

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 현재는 파이어베이스와 연동해서 사용이 불가능합니다.
관련 니즈가 많은 것을 고려해서 내부적으로 방법을 고민하고 있습니다. 
감사합니다.
